<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralDataGUI</class>
 <widget class="QWidget" name="GeneralDataGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informazioni generali sul computo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="priceFieldTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="unitMeasureTab">
      <attribute name="title">
       <string>Unit√† di misura</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QPushButton" name="unitMeasureDelPButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="QTableView" name="unitMeasureView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="unitMeasureAddPButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Campi prezzo</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="3">
        <widget class="QTableView" name="priceFieldTableView"/>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="addPriceFieldPushButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="delPriceFieldPushButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
